README.md

```diff
@@ -246,9 +246,10 @@ conn.on(:stream) do |stream|
     }
 
     # initiate server push stream
+    push_stream = nil
     stream.promise(head) do |push|
-      push.headers({ ... })
-      push.data(...)
```

Comment:
In my testing, this sends the data immediately. I don't see how it sends it later.

```ruby
stream.promise(head) do |push|
       push.headers({ ... })
      push.data(...)
```